README.md

```diff
@@ -15,7 +15,7 @@ The Mobile Center SDK uses a modular architecture so you can use any or all of t
 
 3. **Mobile Center Distribute**: Mobile Center Distribute will let your users install a new version of the app when you distribute it via the Mobile Center. With a new version of the app available, the SDK will present an update dialog to the users to either download or postpone the new version. Once they choose to update, the SDK will start to update your application. This feature will NOT work if your app is deployed to the app store.
 
-4. **Mobile Center Push**: Mobile Center Push enables you to send push notifications to users of your app from our portal. Only Windows Notification Services (WNS) on UWP is supported right now.
+4. **Mobile Center Push**: Mobile Center Push enables you to send push notifications to users of your app from the Mobile Center portal. We use APNS for iOS apps, FCM for Android and WNS for UWP apps. You can also segment your user base based on a set of properties and send them targeted notifications.
```

Comment:
Maybe use the full name instead of initials the first time you mention things like APNS, FCM, and WNS